website_product_configurator/static/src/scss/config_form.scss

```diff
@@ -0,0 +1,155 @@
+#cfg_header {
```

Comment:
Same as the other comment, I recommend wrapping these css rules with a parent element's class, to ensure they aren't leaked outside of this module's feature